### ISSUE TITLE
Stuff for next week

### ISSUE THREAD

**Issue by om4017** [J-Cos/VerdeIslandPassage_Project#5]
Hi @J-Cos I've had some issues already. So I was able to do the NMDS plot with site and fraction. However, I've bought up a summary of the data and I'm not sure I see where the MPA levels that Emma mentioned today. I'm not sure if I've been using the right thing? I've also had some issues creating the rarecurve stuff after lots of googling. 

I think I've shared the code to Github but let me know if it hasn't come up.

In the mean time I will try to get on with the species richness and evenness stuff and see if I get on any better.

Thanks, Liv

**Comment by J-Cos** [comment]
Hi Olivia, I can't see any code here unfortunately.

I think I can answer a few of these points though.

On the MPA levels, I don't know the details about each site, so if you don't already know which is protected maybe ask Emma for that data? There's only 5 site (or something like that) right so it should be simple enough to figure out which is in what type of MPA etc.

Try vegan::rarecure() on the curves. Documentation here: https://rdrr.io/cran/vegan/man/rarefy.html

Try and share some code and then I'll be able to help more!

**Comment by om4017** [comment]
I think I put the code in Verde island passage project and its on here as stuff for next week I think. 

I will ask Emma about the MPA levels for each site and see what she says. It's probably going to be a similar thing but I've basically converted the phyloseq object into a data frame. I then wanted to create a new column called No.houses and one for MPA level and basically create a loop attached to a character string. So every time it detects the string "ART" it will insert in the No.houses column the number of houses I counted for Arthur's reef. I'm going through my notes from weeks ago but any help would be appreciated and also your input on whether you think this is a good idea or not.

I've been playing around with code I've found online to do the rarefaction curve and I keep getting an error message that the phyloseq data can't be found even though I can do plots and estimates for alpha diversity and an NMDS plot so I'm not sure what's going on. 

Am I right that the prune_taxa function is just like rarefying except it maybe wouldn't work on this because there is no taxa allocation yet?

**Comment by J-Cos** [comment]
I've committed some changes to your code. I've made sure everything works and also adjusted the style a bit to hopefully make it easier to read and modify. 

There's probably some new things in there but just read the code carefully and google any functions you don't understand and you'll get it. Try and undertand why your version didn't work and what I've done at each step. If you want to talk through it next week (monday?) let me know.
